docs/examples/eggsample/eggsample/__init__.py

```diff
@@ -0,0 +1,7 @@
+import pluggy
+
+hookimpl = pluggy.HookimplMarker("eggsample")
+"""Marker to be imported and used in plugins"""
+
+pm = pluggy.PluginManager("eggsample")
```

Comment:
should we take a note here that a pluginmanager instance is typically not a global ?

Reply:
Yes, I was not happy with that bit either - I will see, if I can make it a bit more real worl like regarding this, so it is part of some mini config. If this adds too much cruft, I will at least add a comment.

Reply:
I added it in the main and pass it into the class similar to how its done in real world projects now.